.github/workflows/build-images-workflow-run.yml

```diff
@@ -307,6 +316,20 @@ jobs:
       - name: "Push PROD images ${{ matrix.python-version }}:${{ github.event.workflow_run.id }}"
         run: ./scripts/ci/images/ci_push_production_images.sh
         if: matrix.image-type == 'PROD' && steps.defaults.outputs.proceed == 'true'
+      - name: GitHub Checks
```

Comment:
```suggestion
      - name: Add details link to image build job
```

I think?

Reply:
They actually add  Github Checks but yeah. I can make it clearer